Library/Homebrew/cmd/--version.sh

```diff
@@ -2,30 +2,33 @@
 #:
 #:  Print the version numbers of Homebrew, Homebrew/homebrew-core and Homebrew/homebrew-cask (if tapped) to standard output.
 
+# HOMEBREW_CORE_REPOSITORY, HOMEBREW_VERSION are set by brew.sh
+# HOMEBREW_CASK_REPOSITORY is from user enviromnent
```

Comment:
Oops, `HOMEBREW_CASK_REPOSITORY` shouldn't be set by the user! Can you set it in `brew.sh` or in this file with:

```bash
HOMEBREW_CASK_REPOSITORY="${HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-cask"
```

Reply:
OK!, I will set  `HOMEBREW_CASK_REPOSITORY` in `brew.sh` and repair `cmd/--version.sh`.